src/compas/geometry/offset.py

```diff
@@ -138,7 +138,7 @@ def offset_polygon(polygon, distance, tol=None):
     >>> polygon = Polygon([(0.0, 0.0), (1.0, 0.0), (1.0, 1.0), (0.0, 1.0)])
     >>> offsetted_polygon = offset_polygon(polygon, 0.5)
     >>> offsetted_polygon
-    Polygon[[0.5, 0.5, 0.0], [0.5, 0.5, 0.0], [0.5, 0.5, 0.0], [0.5, 0.5, 0.0]]
+    [[0.5, 0.5, 0.0], [0.5, 0.5, 0.0], [0.5, 0.5, 0.0], [0.5, 0.5, 0.0]]
```

Comment:
shouldn't this function return a Polygon object instead of list to align with the input?

Reply:
no, the lower level function shouldn't. the version attached to the polygon class itself should...

Reply:
i see